specification/applicationinsights/ApplicationInsights.LiveMetrics/models.tsp

```diff
@@ -390,7 +390,7 @@ model KeyValuePairStringString {
   key: string;
 
   @doc("Value of the key-value pair.")
-  value: string;
+  value?: string;
```

Comment:
I don't understand this change. I saw your explanation in the PR description (BTW thank you for this -- I see many PRs that lack this kind of explanation):
> This change makes the value property settable

But this should not be dependent on whether the property is required or optional. Making the value property optional means that a key-value-pair that only contains a key is valid -- which makes no sense to me.

Reply:
Discussed with my team, we'll remove this.